.github/workflows/codespell.yml

```diff
@@ -0,0 +1,20 @@
+---
+name: Codespell
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  codespell:
+    name: Check for spelling errors
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: codespell-project/actions-codespell@master
+        with:
+          skip: js,*.svg
+          ignore_words_file: fo,te,als
```

Comment:
this needs to point to a file, see:

- https://github.com/mne-tools/mne-python/blob/4327725e18a87e6fa3c75c637fa9297d053d95ed/.github/workflows/codespell_and_flake.yml#L42
- https://github.com/mne-tools/mne-python/blob/main/ignore_words.txt

Not sure if it's possible to just list the words here in the yml file :)

Reply:
My wrong, it's `ignore_words_list` instead of `ignore_words_file`. See [codespell-project/actions-codespell](https://github.com/codespell-project/actions-codespell).

As long as it's a handful small variable names, I prefer avoiding an additional file.